shard-operator-integration-tests/src/test/resources/features/bridgeingress-deploy.feature

```diff
@@ -0,0 +1,50 @@
+Feature: BridgeIngress deploy and undeploy
+
+  Background:
+    Given create Namespace
+
+  # Using "dummy" image as real BridgeIngress image requires Kafka
+  Scenario: BridgeIngress is in phase AVAILABLE
+    When deploy BridgeIngress:
+    """
+    apiVersion: com.redhat.service.bridge/v1alpha1
+    kind: BridgeIngress
+    metadata:
+      name: my-bridge-ingress
+      app.kubernetes.io/managed-by: bridge-fleet-shard-operator
+    spec:
+      image: nginx:1.14.2
```

Comment:
That's interesting. I think we might need to think in a way to enforce a "real" image for our custom resources. 🤔

Reply:
My plan is to have two levels of tests.
First one is Shard operator tests (integration) checking interactions between operator and Kubernetes environment. These should be ideally quick tests with scope limited to operator only, possibly mocking unnecessary dependencies. Assuming that the dependencies themselves are tested in their dedicated tests.
Second level are e2e tests (user acceptance) testing whole solution. These tests will be significantly slower as they will require full environment with all dependencies. The tests there will be written from user perspective (checking outputs visible to user).